src/exceptions.py

```diff
@@ -0,0 +1,24 @@
+list_ValueError = []
+list_KeyError = []
+list_ZeroDivisionError = []
+
+
+def add_error_v(v: ValueError) -> None:
```

Comment:
Все хорошо но переименуй вот эти функции в полные имена

```suggestion
def add_value_error(v: ValueError) -> None:
```